test/cypress/e2e/login-logout.cy.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress" />
+// Use `cy.dataCy` custom command for more robust tests
+// See https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements
+
+// ** This file is an example of how to write Cypress tests, you can safely delete it **
+
+// This test will pass when run against a clean Quasar project
```

Comment:
```suggestion
```

@stravid87, you can you suggest with commit doing this.
It will show a comparison and it will be possible to confirm directly from the comment.

Just drag the lines you want to change or remove and click **Add a suggestion**.
(as I did when adding this comment)

<img width="652" alt="Captura de Tela 2023-03-28 às 17 26 53" src="https://user-images.githubusercontent.com/26576201/228358848-f52ccc63-eb9f-4ece-8ee4-a30442591b73.png">